packages/agent-graph/src/hooks/useGraphCamera.ts

```
@@ -23,14 +25,23 @@ export interface UseGraphCameraResult {
   zoomIn: () => void;
   zoomOut: () => void;
   updateInertia: () => void;
 }
 
 export function useGraphCamera(): UseGraphCameraResult {
-  const transformRef = useRef<CameraTransform>({ x: 0, y: 0, zoom: 1 }) as React.MutableRefObject<CameraTransform>;
+  const transformRef = useRef<CameraTransform>({
+    x: 0,
+    y: 0,
+    zoom: 1,
+  }) as React.MutableRefObject<CameraTransform>;
```

ESLint

[error] 'MutableRefObject' is deprecated.

(sonarjs/deprecation)